CHANGELOG.md

```diff
@@ -2,12 +2,13 @@
 
 All notable changes to this project will be documented in this file.The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
-## [0.1.0](https://github.com/puppetlabs/ruby-pwsh/tree/0.1.0) (2019-10-25)
+## [0.2.0](https://github.com/puppetlabs/ruby-pwsh/tree/0.2.0) (2019-11-25)
```

Comment:
This looks odd.  Shouldn't this be adding a version to the changelog, not replacing it?

Reply:
The original entry was actually post 0.1.0, but I didn't notice; this is an artifact of having used changelog generator after a release, note line 7 was comparing `0.1.0` to `0.1.0` prior to this changeset.

CLG should really only be run _at_ release time, having run it between releases (changelog generator was added after the initial release) has created this oddity.

Reply:
Errrr ok... I guess that's fine?

Reply:
Does this mean the changelog for 0.1.0  no longer exists? That seems weird, but oh well.

Reply:
There never was one, the prior changelog should've been categorized as unreleased.

I'm not sure how to have it standardize a message for 0.1.0 as Initial release. /shrug